src/scraper/tests/test_scraper.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import jsonpickle
+import vcr
+from scraper.scraper import scrap
+from common.scraperstructs import Document
+import os
+
+class ScraperTests(unittest.TestCase):
+
+    # to record:
+        # set disconnected=False (to set 2sec delay between requests)
+        # delete test_run_scraper.yaml
+        # set record_mode='once'
+    def test_run_scraper(self):
+        nb_doc = 20  # 20 to keep test under 2sec
+        curr_doc = 0
+
+        directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
integration test for scraper, use vcrpy (register then replay html requests to not make tests reliables)